app-service-template/res/configuration.toml

```diff
@@ -105,11 +112,20 @@ TokenFile = "/tmp/edgex/secrets/new-app-service/secrets-token.json"
   Port = 59881
 
   [Clients.core-command]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59882
-
-  [Clients.support-notifications]
+#  Protocol = "http"
+#  Host = "localhost"
+#  Port = 59882
+  # Choose to use new messaging based communication for Command Client.
+  # Only valid if MessageBus Trigger is used
+  # If prefer REST, remove below and uncomment above
+  UseMessageBus = true
+  [Clients.core-command.Topics]
+  QueryRequestTopic = "edgex/core/commandquery/request"
```

Comment:
Do you think we need to change this to `QueryRequestTopicPrefix` to be aligned with `CommandRequestTopicPrefix` ?

Reply:
> Do you think we need to change this to `QueryRequestTopicPrefix` to be aligned with `CommandRequestTopicPrefix` ?

Yes, for consistency since it is a prefix.

Reply:
PR created: https://github.com/edgexfoundry/go-mod-messaging/pull/182